cont.c

```diff
@@ -1618,11 +1618,13 @@ root_fiber_alloc(rb_thread_t *th)
     return fib;
 }
 
+rb_fiber_t ruby_storage_root_fiber = {0};
```

Comment:
The initializer isn't needed just for zero-fill.

Reply:
Correct, zero'd out implicitly in the data segment, no need to be explicit with zero-fill.